packages/web3-core-method/src/index.js

```diff
@@ -56,6 +56,7 @@ var Method = function Method(options) {
     this.transactionBlockTimeout = options.transactionBlockTimeout || 50;
     this.transactionConfirmationBlocks = options.transactionConfirmationBlocks || 24;
     this.transactionPollingTimeout = options.transactionPollingTimeout || 750;
+    this.blockHeaderTimeout = options.blockHeaderTimeout || 5; // 10 seconds
```

Comment:
```suggestion
    this.blockHeaderTimeout = options.blockHeaderTimeout || 5; // 5 seconds
```

Reply:
I'm just going to commit this since it's a comment change, and so I can provide my approval without it going stale